src/main/java/net/mcreator/element/converter/v2023_4/EntityGlowLayerConverter.java

```diff
@@ -47,6 +47,7 @@ public GeneratableElement convert(Workspace workspace, GeneratableElement input,
 					glowLayer.setWorkspace(workspace);
 					glowLayer.model = "Default";
 					glowLayer.texture = glowTexture;
+					glowLayer.hurtAnimation = true;

```

Comment:
The reason for
> 1. Should the parameter be inverted?

Current naming would probably also need a converter to set it to true for existing layers (like with `requiresCorrectToolForDrops`).

Reply:
disableHurtAnimation?

Reply:
This is more of an overlay than animation though given how MC calls this

Reply:
Makes sense.

Reply:
✔️ Fixed